docs/snippets/package/aws-with-vpc/crossplane.yaml

```diff
@@ -11,4 +26,4 @@ spec:
     version: ">=v1.4.0-0"
```

Comment:
I think I would update this to the latest crossplane release too - no reason to encourage the use of older versions

Reply:
@bobh66 Yeah, I thought this was outdated as well, but latest might be too restrictive -- I was testing on a hosted Upbound control plane for convenience, which was running 1.8.1 so I think some slack feels reasonable. @negz any guidance here on what this should be?

Reply:
Thanks - LGTM